<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorProfilePage</class>
 <widget class="QWidget" name="DoctorProfilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医生个人信息</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::WrapAllRows</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>姓名:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>科室:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="departmentInput">
       <item>
        <property name="text">
         <string>内科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>外科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>妇产科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>儿科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>眼科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>耳鼻喉科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>口腔科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>皮肤性病科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>精神心理科</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>肛肠科</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>工号:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="employeeNoInput"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>职称等级:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="titleInput"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>详情信息:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="detailInput"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>上班时间:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="workingTimeLayout">
       <item>
        <widget class="QComboBox" name="startHourCombo"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="startMinuteCombo"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string> 到 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="endHourCombo"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="endMinuteCombo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>挂号费用:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="registrationFeeInput"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>单日患者上限:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="dailyLimitInput"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>是否休假:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="vacationLayout">
       <item>
        <widget class="QRadioButton" name="vacationYes">
         <property name="text">
          <string>是</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="vacationNo">
         <property name="text">
          <string>否</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="emailInput"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>电话:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="phoneInput"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>编辑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
